frontend/src/App.vue

```diff
@@ -2,7 +2,6 @@
     v-app
         PageHeader
         v-content
-            v-container(fluid)
```

Comment:
Why remove this?

Reply:
For full coverage of the background image.. default padding of the container  was 12px  hence the background was not covering the entire space..